Demo/Demo/ViewModels/PayPalVaultViewModel.swift

```diff
@@ -0,0 +1,66 @@
+import UIKit
+import PayPalWebPayments
+import CorePayments
+
+class PayPalVaultViewModel: VaultViewModel, PayPalVaultDelegate {
+
+    @Published var paypalVaultToken: PayPalVaultResult?
+
```

Comment:
I considered having same VaultResult for card and PayPal vaulting but UpdateSetupToken endpoint
for card vaulting returns a lot more values, so I opted to keep them separate. If only minimum values
that I display here are needed, they can be consolidated into a single VaultResult and VaultView and 
included in the VaultState.

Reply:
From my perspective, communicating the viewModel with the view without using the `State` structure somewhat breaks the paradigm. For now, I think it's better to move PayPalVaultViewModel.`paypalVaultTokenResponse` and CardVaultViewModel.`updateSetupTokenResponse` back to `VaultState`, as it was in previous commits. Maybe someone else on the team could provide some input. Considering that the purpose of the Demo app is all about internal testing, showcase functionalities,  and providing merchants with examples of SDK usage

Reply:
Yes, I thought I might get comment that it's confusing to be able to access updateSetupToken from PayPalViewModel but I prefer them all in vaultState as well.